saleor/graphql/product/types/products.py

```diff
@@ -832,13 +832,6 @@ def resolve_tax_type(root: models.ProductType, info):
         tax_data = info.context.plugins.get_tax_code_from_object_meta(root)
         return TaxType(tax_code=tax_data.code, description=tax_data.description)
 
-    @staticmethod
-    def resolve_tax_rate(root: models.ProductType, _info, **_kwargs):
```

Comment:
We drop the resolver, but the field is still available in the type, is that what we want, or we should remove the field at all?

Reply:
We should remove the field.

Reply:
But this field is a public API (no staff permission required) and it wasn't deprecated before. We should deprecate it first in Saleor 2.11. We also need to mention it in the changelog as a breaking change.